CougSat1-Ground/src/space/cougs/ground/gui/subsystems/modules/AttitudeIndicator.java

```diff
@@ -0,0 +1,155 @@
+package space.cougs.ground.gui.subsystems.modules;
+
+import java.awt.Dimension;
+import java.awt.FontMetrics;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.Insets;
+import java.awt.RenderingHints;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+
+import javax.imageio.ImageIO;
+import javax.swing.BorderFactory;
+import javax.swing.JPanel;
+
+import space.cougs.ground.gui.UIScaling;
+import space.cougs.ground.gui.utils.CustomColors;
+import space.cougs.ground.gui.utils.Fonts;
+
+public class AttitudeIndicator extends JPanel implements UIScaling {
+
```

Comment:
Copied from Map class. Remove code not applicable for AttitudeIndicator and put a //TODO comment